adding note about metrics-server to pages/ksphere/kommander/1.2/clust… (#3288)
mesosphere/dcos-docs-site: pages/ksphere/kommander/1.2/clusters/attach-cluster/attach-eks-cluster/index.md

   kubectl get no
   ```

> 1. The metrics that Kommander displays on its cluster view don't come from Prometheus. Rather they come from the Kubernetes metrics API. In order for this API to be available, it must be exposed by a [metrics server][k8s-metrics-server]. In order to do that, you must [deploy the metrics server][deploy-metrics-server], otherwise the EKS cluster may fail to stay attached. To deploy the server, run the following command:

Reviewer: ```suggestion
1. The metrics Kommander displays on its cluster view do not come from Prometheus. They come from the Kubernetes metrics API. You must expose this API using a [metrics server][k8s-metrics-server] to make it available. To do that, you must [deploy the metrics server][deploy-metrics-server], otherwise the EKS cluster may fail to stay attached. To deploy the server, run the following command:
```
